client/src/components/CKANRegistryDisplay.js

```diff
@@ -117,25 +124,54 @@ class CKANRegistryDisplay extends Component {
    */
   getVisibleFields() {
     return this.props.fields
-      .filter(field => parseInt(field.ShowInResultsView, 10) === 1)
+      .filter(field => field.ShowInResultsView)
       .map(field => field.OriginalLabel);
   }
 
+  /**
+   * Finds all visible fields that have configured "ResultConditions", e.g. fields
+   * that should only show data in them that match certain conditions. When found,
+   * these will be applied as query filters before the data is loaded.
+   *
+   * @param {Query} query
+   */
+  applyDefaultFilters(query) {
+    const { fields } = this.props;
```

Comment:
This method is can have the call trace: `applyDefaultFilters <- resetQueryFilters <- constructor`. Is accessing props going to be a problem here? I can't remember whether it's safe to access props like this in the constructor in React components.

Reply:
Did my research. It's fine as long as you `super(props)` first. Probably should've guessed that.